.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```diff
@@ -0,0 +1,108 @@
+name: "Report a Linux Issue"
+description: "Create a report for a runtime related Linux Issue"
+labels: ["os:linux"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
+           - Linux Distribution and Version
+           - Source of the mpv binary
+           - If known which version of mpv introduced the problem
+           - Window Manager and version
+           - GPU model, driver and version
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - Linux version:
+          - Source:
```

Comment:
Source seems vague here, maybe `Source of binary:`?

Reply:
this is only the shorthand prefilled list for filling out the issue template. the instructions above mention "- Source of the mpv binary" already.